src/acl.c

```diff
@@ -1990,28 +2286,78 @@ void aclCommand(client *c) {
         user *u = ACLGetUserByName(username,sdslen(username));
         if (u) ACLCopyUser(tempu, u);
 
+        /* Initial pass of client args to collect selectors */
+        sds *acl_args = (sds *) zmalloc(sizeof(sds) * c->argc);
+        int new_argc = 0;
+
+        /* Selector definitions should be sent as a single argument, however
+         * we will be lenient and try to find selector definitions spread 
+         * across multiple arguments. */
```

Comment:
i'm not sure i understand it, and if i do, i'm not sure it's necessary or desired to support that.
maybe after i read the test code i'll understand better.

Reply:
I'll add a top comment for this.